packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -2390,6 +2376,87 @@ contract GenArt721CoreV3_Engine is
         );
     }
 
+    /**
+     * @notice Initializes the contract with the provided `engineConfiguration`.
+     * This function should be called atomically, immediately after deployment.
+     * Only callable once. Validation on `engineConfiguration` is performed by caller.
+     * @param engineConfiguration EngineConfiguration to configure the contract with.
+     * @param adminACLContract_ Address of admin access control contract, to be
+     * set as contract owner.
+     * @param defaultBaseURIHost Base URI prefix to initialize default base URI with.
+     * @param bytecodeStorageReaderContract_ Address of bytecode storage reader contract.
+     */
+    function _initialize(
```

Comment:
moved initialize to internal function - something we want to implement before #1576 for better inheritance properties